android/src/com/mapswithme/maps/MwmActivity.java

```diff
@@ -1426,9 +1453,12 @@ private void showMainMenu(boolean show)
   @Override
   public void onRoutingPlanStartAnimate(boolean show)
   {
-    int totalHeight = calcFloatingViewsOffset();
-    adjustCompassAndTraffic(!show ? UiUtils.getStatusBarHeight(getApplicationContext())
-                                  : totalHeight);
+    int offset;
```

Comment:
```suggestion
    int offset = 0;
```
> Local variables are slightly different; the compiler never assigns a default value to an uninitialized local variable. If you cannot initialize your local variable where it is declared, make sure to assign it a value before you attempt to use it. Accessing an uninitialized local variable will result in a compile-time error.

Source: https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html